src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -1203,7 +1204,6 @@ export class TableComponent implements AfterViewInit, OnInit, OnChanges, OnDestr
       let rows = this.columnFilters.length !== 0 ? this.doColumnFiltering() : this.data;
 
       if (this.search.length > 0 && rows?.length) {
-        this.expanded = undefined;
```

Comment:
Note to @ivoalmeida: this was causing the expanded row to collapse automatically when the search field has some value. Saw this mostly in the configuration page but other pages are also affected.